source/DEVS-Network/DEVSTestNetwork.class.st

```diff
@@ -0,0 +1,26 @@
+Class {
+	#name : 'DEVSTestNetwork',
+	#superclass : 'DEVSCoupledComponent',
+	#category : 'DEVS-Network',
+	#package : 'DEVS-Network'
+}
+
+{ #category : 'modeling' }
+DEVSTestNetwork >> initialize [ 
+	| serverName node server |
+	super initialize.
+	
+	node := DEVSNetworkNode new name: #networknode.
+	self addComponent: node.
+	1 to: 3 do: [ :n |
+		serverName := ('server', n asString).
+		server := DEVSUDPServer new 
+			name: serverName;
+			address: ('10.0.0.', n asString).
+		self addComponent: server.
+		node addOutputPortNamed: serverName.
```

Comment:
Is it necessary to have name of output port of network node same as server name? Maybe it needs just below couplings to be not server name specific? But I'm not fully follow, what you need to achieve and how to wire node and servers). Full broadcast?

Reply:
No, it should be unicast. And the server name is just a hack to make it work. I wrote that in my comment that this is one thing to solve. In a usual network the UDP server would send a message to server3. Then the server3 would be resolved to its ip address and a message created with the ip address of server3 instead so that the network layer has not to deal with it.